Scripts/Interfaces/ILinkedTool.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+/// <summary>
+/// Provides access to other tools of the same type
+/// </summary>
+public interface ILinkedTool
+{
+	/// <summary>
+	/// List of other tools of the same type (not including this one)
+	/// </summary>
+	List<ILinkedTool> otherTools { get; }
```

Comment:
Actually, I'm starting to think that this goes for more than just tools. We could use this in MiniWorldWorkspace to move some of the logic out of EditorVR.MiniWorlds